### DISCUSSION TITLE
General recommendation for recipes

### DISCUSSION THREAD

**Discussion by satelin2002** [chakra-ui/panda#788] [General]
what is the general recommendation for recipes? 

[Atomic Recipe (or cva)](https://panda-css.com/docs/concepts/recipes#atomic-recipe-or-cva)
or 
[Config Recipe](https://panda-css.com/docs/concepts/recipes#config-recipe)

like for buttons, select, file and so on..

and similarly for components like Card,Accordian and so on?

**Comment by anubra266** [comment]
Atomic recipes as the name implies are for simpler usecases, simple components that you might need in many places. An example is when you're trying to avoid dynamic styling. Basically components that are not defined in the design system.
For other stuff; use config recipe. Especially when you know the component is a part of your design system.
All the components you mentioned, I'd do them with config recipes; from button to accordion.

**Reply by pawelblaszczyk5** [reply]
However, the config recipe contributes more towards the bundle size, because they aren't atomic. Shouldn't we use the cva recipe, unless we need the goodies from config recipe (conditional/responsive variants)? 🤔
